Source/Microsoft.Gateway/Microsoft.InnerEye.Listener.Wix.Actions/CustomActions.cs

```diff
@@ -53,89 +52,84 @@ public static ActionResult ValidateProductKey(Session session)
             Debugger.Launch();
 #endif
 
+            // Make sure that the applications are run as services.
+            var gatewayProcessorConfigProvider = new GatewayProcessorConfigProvider(
+                null,
+                ConfigInstallDirectory);
+            gatewayProcessorConfigProvider.SetRunAsConsole(false);
+
+            var gatewayReceiveConfigProvider = new GatewayReceiveConfigProvider(
+                null,
+                ConfigInstallDirectory);
+            gatewayReceiveConfigProvider.SetRunAsConsole(false);
+
             // Check if the installer is running unattended - lets skip the UI if true
             if (session.CustomActionData[UILevelCustomActionKey] == "2")
             {
                 return ActionResult.Success;
             }
 
-            var gatewayProcessorConfigProvider = new GatewayProcessorConfigProvider(
-                null,
-                ProcessorInstallDirectory);
-
-            var processorSettings = gatewayProcessorConfigProvider.ProcessorSettings();
+#pragma warning disable CA5364 // Do Not Use Deprecated Security Protocols
+#pragma warning disable CA5386 // Avoid hardcoding SecurityProtocolType value
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12;
```

Comment:
Why are we doing this?

Reply:
Because the installer seems to run with just SSL3 and TLS1.0. I have added a comment and enabled just TLS1.2